paymentsheet/src/main/java/com/stripe/android/paymentsheet/MandateHandler.kt

```diff
@@ -53,7 +53,7 @@ internal class MandateHandler(
                 coroutineScope = viewModel.viewModelScope,
                 selection = viewModel.selection,
                 merchantDisplayName = viewModel.config.merchantDisplayName,
-                isVerticalMode = viewModel.config.paymentMethodLayout == PaymentMethodLayout.Vertical,
+                isVerticalMode = viewModel.config.paymentMethodLayout != PaymentMethodLayout.Horizontal,
```

Comment:
How will this work if we start using Automatic to mean Horizontal or Vertical depending on e.g. the number of PMs? I think we may end up needing an internal type which is only Vertical/Horizontal, so we don't have to check at each callsite whether we should be in horizontal or vertical. 

I think this works for now, and can be figured out when we actually need to, but just thinking out loud

Reply:
Yup! We'll need a mechanism to not reference these directly. I'm thinking it'll be something along the lines of a method that takes in the required params, and returns a boolean for horizontal/vertical.